python/common/org/python/types/Str.java

```diff
@@ -287,6 +287,9 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
         else if (other instanceof org.python.types.Float) {
             throw new org.python.exceptions.TypeError("can't multiply sequence by non-int of type '" + other.typeName() + "'");
         }
+        else if (other instanceof org.python.types.Str) {
+            throw new org.python.exceptions.TypeError("can't multiply sequence by non-int of type '" + other.typeName() + "'");
+        }
```

Comment:
Is there any reason that this can't just be else { raise TypeError }? Are there any error messages here other than "can't multiply sequence by non-int of type ..."? This might mean we end up with a bunch of other passing multiply tests as a side effect.